static/js/apps/biomed_nl/app.tsx

```diff
@@ -52,16 +60,18 @@ interface DisplayedAnswer {
   debugInfo: string;
 }
 
+// Eneable dropdown for Footer and Debug sections
```

Comment:
nit: enable

nit: this function doesn't actually enable the dropdown, it's just the component for the always displayed part of the collapsible

Reply:
updated, hopefully this makes more sense?